### PR TITLE
New Cheat Sheet: CSS Flexbox

### PR DESCRIPTION
**Pull request by karunk**
- [ ] New Instant Answer
  - [x] Cheat Sheets: **`New CSS Flexbox Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**

---

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @karunk

Instant Answer Page: https://duck.co/ia/view/css_flexbox_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@karunk Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by akanshgulati** [comment]
@karunk Please update the description of this cheatsheet as it has wrong spelling. Thanks

**Comment by karunk** [comment]
@akanshgulati Thanks for pointing out. I have updated.

File: share/goodie/cheat_sheets/json/flexbox.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            "key": "column;"
+        },{
+            "val": "Right to left in ltr; left to right in rtl.",
+            "key": "row-reverse;"
+        },{
+            "val": "Same as row-reverse but bottom to top.",
+            "key": "col-reverse;"
+        }],
+        "Justify-Content": [{
+            "val": "This defines the alignment along the main axis.",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/flexbox.json]: lets remove periods from end of val fields

**Comment by pjhampton** [comment]
Thanks for this @karunk 👍

**Comment by daxtheduck** [comment]
@karunk this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**